client.go

```diff
@@ -4,11 +4,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/Piszmog/cfservices"
-	"golang.org/x/oauth2/clientcredentials"
```

Comment:
Please revert this. For this project, import formatting uses `gofmt` instead of `goimports`. 

Hopefully imports formats get figured out (see [#37719](https://github.com/golang/go/issues/37719))